linetools/lists/linelist.py

```diff
@@ -760,9 +810,26 @@ def available_transitions(self, wvlims, n_max_tuple=None, min_strength=0.):
         else:
             raise SyntaxError('min_strength must be a float value')
 
+        # Cut on wavelength first!
+        gdwv = (self.wrest>wvlims[0]) & (self.wrest < wvlims[1])
+        if np.any(gdwv):
+            tmp_data = self._data[gdwv]
+            extras = self._extra_table[gdwv]
+            data = hstack([tmp_data,extras])
+        else:
+            return None
+
+        # Sort
+        data.sort(['ion_name','rel_strength'])
+        data.reverse()
+
+
         # Identify unique ion_names (e.g. HI, CIV, CIII)
-        unique_ion_names = np.unique(self._data['ion_name'])
+        # This is WRONG for FeII*
+        unique_ion_names, idx = np.unique(data['ion_name'], return_index=True)
```

Comment:
A unique naming convention will help with this kind of things.

Reply:
well, I think the proper thing is to use Z, ion, and Eij